utils/test-utils/src/main/java/de/sovity/edc/extension/e2e/connector/config/ConnectorConfigFactory.java

```diff
@@ -36,9 +37,39 @@ public class ConnectorConfigFactory {
 
     public static ConnectorConfig forTestDatabase(String participantId, TestDatabase testDatabase) {
         val firstPort = getFreePortRange(5);
-        var config = basicEdcConfig(participantId, firstPort);
-        config.setProperties(configureDatasources(testDatabase.getJdbcCredentials()));
-        return config;
+
+        // The initialization of the Map is split into several statements
+        // due to the parameter limit of Map.of(...)
+        var propertiesInput = new HashMap<>(Map.of(
+            ConfigProps.MY_EDC_NETWORK_TYPE, ConfigProps.NetworkType.UNIT_TEST,
+            ConfigProps.MY_EDC_FIRST_PORT, String.valueOf(firstPort),
+            ConfigProps.EDC_API_AUTH_KEY, "api-key-%s".formatted(UUID.randomUUID().toString()),
+            ConfigProps.MY_EDC_C2C_IAM_TYPE, "mock-iam",
```

Comment:
I'm not sure how I like to have this one set here as I have tests that really don't want the mock auth in place and this would now be the default that I have to unset.